lisp/ess-julia.el

```diff
@@ -281,19 +71,19 @@ VISIBLY is not currently used."
       (kill-buffer)
       (nreverse out))))
 
-(defvar julia--manual-topics nil)
-(defun julia-manual-lookup-function (&rest args) ; args are not used
+(defvar ess-julia--manual-topics nil)
+(defun ess-julia-manual-lookup-function (&rest args) ; args are not used
   (interactive)
   "Look up topics at http://docs.julialang.org/en/latest/manual/"
   ;; <li class="toctree-l1"><a class="reference internal" href="introduction/">Introduction</a></li>
-  (let* ((pages (or julia--manual-topics
-                    (setq julia--manual-topics
-                          (julia--retrive-topics "http://docs.julialang.org/en/latest/manual/"))))
+  (let* ((pages (or ess-julia--manual-topics
+                    (setq ess-julia--manual-topics
+                          (ess-julia--retrive-topics "http://docs.julialang.org/en/latest/manual/"))))
```

Comment:
Hmm.... you mean to use help in julia's  ess-integrated mode one would access the internet ?

No way now,.... maybe in ten years.... but honestly I'd even hate it then.
Hej, this is free software so with free docs...  and that free means I want the relevant stuff on my hard disk.